code/game/gamemodes/clock_cult/clock_mobs.dm

```diff
@@ -0,0 +1,279 @@
+/mob/living/simple_animal/hostile/anima_fragment //Anima fragment: Low health but high melee power. Created by inserting a soul vessel into an empty fragment.
+	name = "anima fragment"
+	desc = "An ominous humanoid shell with a single glowing yellow eye, pushed into the air by jets of blazing yellow fire."
```

Comment:
I suggest "An ominous shell with what appears to be a rapidly spinning cogwheel for a head, floating on a jet of flickering reddish fire."